src/lib/components/consignments/__stories__/consignments-todo.story.tsx

```diff
@@ -30,38 +39,32 @@ const metadata: ConsignmentMetadata = {
   displayString: "5/5",
 }
 
-const withMetadata: ConsignmentSetup = {
+const withMetadata = {
   ...withPhotos,
   metadata,
 }
 
-const withLocation: ConsignmentSetup = {
+const withLocation = {
   ...withMetadata,
   location: "Huddersfield, UK",
 }
 
-const withProvenance: ConsignmentSetup = {
+const withProvenance = {
   ...withLocation,
   provenance: "This work has seen many hands.",
 }
```

Comment:
The one potential problem with this is that if there are optional props in the interface, then you’re allowed to misspell them here and it won’t get noticed, because `any` is considered a superset of `BottomAlignedProps` and thus may contain more props.